bgp/bgp.proto

```diff
@@ -40,6 +45,55 @@ message ClearBGPNeighborRequest {
     SOFT = 0;         // Send route-refresh and reapply policy.
     SOFTIN = 1;       // Re-apply inbound policy on stored Adj-RIB-In.
     HARD = 2;         // Teardown and restart TCP connection.
```

Comment:
Deprecate the HARD field and add description such as:

```suggestion
    // The HARD Mode is intended to teardown and restart the BGP session
    // TCP connection. This field does not define what the behavior is regarding
    // BGP graceful restart. The HARD_RESET and GRACEFUL_RESET fields 
    // should be used instead of HARD.
    HARD = 2;      [deprecated=true];
```